README.md

```diff
@@ -1,2 +0,0 @@
-# Week11-hangman
-Homework: a node-based hangman game with recursive functions.
```

Comment:
3. you either forgot to stage the Word file in this commit or GH truncated it because your PR is too big (read #1)